### DISCUSSION TITLE
Useful links and videos discussion

### DISCUSSION THREAD

**Discussion by glazet** [glazet/QuadEVCharger#2] [Ideas]
Here is a place to share useful links and videos to aid us in the project.

**Comment by glazet** [comment]
Very useful video on communication between EVSE and EV.

https://www.youtube.com/watch?v=CqE6O8PGjMo&ab_channel=cethalassery

**Comment by glazet** [comment]
This webinar is a good resource for our who are customers could be as well as regulations and standards for EV chargers in residential locations. 

https://www.youtube.com/watch?v=jtcR62GbA1c&ab_channel=Veloz

**Reply by neilh10** [reply]
This Veloz webinar is excellent for poising the  problems, but not the solutions. From 17min, they do reference shared charging  - but only duo charger. The current solutions are penalties for leaving the vehicle attached after it is charged - $25

**Comment by neilh10** [comment]
Good to see some links, though while listening to a video can be educational, it needs some distillation. Can you summarize what its about, why they are authoritative, and what is of potential value to the project.

A colleague referred this link to me of a recent PGE conference,
Residential Charging (including Multi-Family): https://www.youtube.com/watch?v=yTF9JkeayMM - PGE is admitting they are having trouble processing new connection requests. I've also heard of this locally, that an upgraded panel for a business took a year - pretty much untenable situation.

my synthesis is that for a new multifamily build, applying for a permit for PGE EV Ready for 4 parking spaces is going to be processed faster than if the multifamily developers decides to apply for EV Ready socket per parking space per units meter. CALGREEN EV Ready socket per four parking spaces is only a minimum requirement, a developer can if they see fit upgrade. However if this delays the project build as PGE can't support it, then developer is unlikely to go down that route.

which is another point in the business' case for a QUAD EV Charger

A potential customer question might be - For a multifamily build, say greater than 25 units, do they think PGE will process (provide electric connection) for a CALGREEN requirement for one EV ready socket per 4 parking spaces, faster than an EV Ready Socket per parking space tied to a Multifamily units meter.
It does require some understanding of the CALGREEN EV ready requirements

Another link in here is the PGE Innovation summit 2023
https://www.pge.com/en_US/about-pge/environment/what-we-are-doing/innovative-community-programs/innovation-technology-strategy.page

**Comment by djangodemetri** [comment]
[Making PWM with RPi](https://www.electronicwings.com/raspberry-pi/raspberry-pi-pwm-generation-using-python-and-c)
[Type 2 Charging Communication](https://www.picoauto.com/library/automotive-guided-tests/charger-vehicle-communications-type-2/)
[Another Communication Link](https://www.vector.com/us/en/know-how/smart-charging/communication-protocols/#c236816)
[More Technical Documentation (Hopefully)](https://www.vector.com/us/en/company/get-info/order-info-material/posters/)
[Slide 7 Shows Actual 1KHz signal](https://www.energy.gov/eere/vehicles/articles/vehicle-technologies-office-merit-review-2014-vehicle-communications-and)
[Video on 1KHz and Its Changes](https://www.youtube.com/watch?v=pjHx5zQPetg)
[Simulating an EV Plugging Into a EVSE](https://www.youtube.com/watch?v=kqkEr93NjDg) 
[Specifics of PWM on Page 41](https://hmr.araiindia.com/Control/AIS/210201752832PMAIS138_Part1.pdf)
[Example of Arduino Powered System](https://www.instructables.com/Arduino-EV-J1772-Charging-Station/)
[How Surge Protectors Work](https://www.cablewholesale.com/support/technical_articles/the_mysteries_of_surge_protectors.php)
[Presentation Including Regulations, Self Built Circuit, Info about OpenEVSE](https://www.fveaa.org/fb/J1772_386.pdf)
[Forums for OpenEVSE](https://groups.google.com/g/OpenEVSE)
[GitHub for OpenEVSE](https://github.com/lincomatic/open_evse)
